services-matching/src/test/java/com/mapbox/api/matching/v5/MapboxMapMatchingTest.java

```diff
@@ -599,7 +599,8 @@ public void testRouteOptionsApproaches() throws Exception {
     Response<MapMatchingResponse> response = mapMatching.executeCall();
     RouteOptions routeOptions = response.body().matchings().get(0).routeOptions();
 
-    assertEquals("unrestricted;curb", routeOptions.approaches());
+    assertEquals("unrestricted", routeOptions.approachesList().get(0));
```

Comment:
To keep previous test, wouldn't `FormatUtils#formatApproaches` passing `routeOptions.approachesList()` work here and return `"unrestricted;curb"`?

Reply:
will add a new one